### PR TITLE
Day10

### PR DESCRIPTION
**Pull request by JustineSara**

### CODE REVIEW

File: src/t/d10.clj [hunk 1]
```diff
@@ -0,0 +1,85 @@
...
+96549874
+45678903
+32019012
+01329801
+10456732")
+
+(defn parse-input
+  [input]
+  (->> (cljstr/split-lines input)
+       (map vec)
```
Comment (gaverhae) [inline src/t/d10.clj:21]: I don't think this line does anything. It turns the per-line strings into vectors of characters, but you then `map-indexed` over them anyway. `(map-indexed f str)` will do the same as `(map-indexed f (vec str))`.

File: src/t/d10.clj [hunk 2]
```diff
@@ -0,0 +1,85 @@
...
+       (map-indexed
+         (fn [y l]
+           (->> l
+           (map-indexed
+             (fn [x c] {(parse-long (str c)) #{[x y]}})
+             )
+           (apply merge-with into))))
+       (apply merge-with into)
+  ))
```
Comment (gaverhae) [inline src/t/d10.clj:30]: This is like the fourth time this year already. Makes me wonder if we should make a "text to grid" library function. 🤔

**Review by gaverhae — APPROVED** [review]

File: src/t/d10.clj (hunk 1 above)
Comment (JustineSara) [reply src/t/d10.clj:21]: yes, at the start I thought I was going to just keep a vector of vector of the characters and build from there, and then I decided I wanted another structure >.>
forgot to clean-up again

File: src/t/d10.clj (hunk 2 above)
Comment (JustineSara) [reply src/t/d10.clj:30]: if you do it too, it could fit nicely into your library.

**Pull request merged**